support/client/lib/vwf/view/kineticjs.js

```diff
@@ -908,12 +908,14 @@ define( [ "module", "vwf/view", "jquery", "vwf/utility", "vwf/utility/color", "v
 
             // Recurse through children to refresh the ones using hitgraph from cache
             var children = kineticObj.children;
```

Comment:
I'm a fan of the js adaptation of the [Null Object Pattern](https://en.wikipedia.org/wiki/Null_object_pattern), where we ensure that the value we have is valid to avoid needing checks later on (It often leads to easier to parse code with fewer indentations).  In that pattern, you could do something like this:

    var children = kineticObj.children || [];

... then your original code would be fine.

Reply:
Thanks for the suggestion.  Implementing that change.